_storage/hub-operation.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: storage
+permalink: /:collection/:path.html
+---
+# Understand hub operation
+{:.no_toc}
+
+This page describes the considerations hub operators must take into account when creating and operating a Gaia storage hub.
+
+* TOC
+{:toc}
+
+
+## Configuration files
+
+You should store a JSON configuration file either in the top-level directory of
+the hub server. Alternatively, you can specify a file location using the
+`CONFIG_PATH` environment variable.  The following is an example configuration file for Amazon S3:
+
+```json
+{
+  "servername": "localhost",
+  "port": 4000,
+  "driver": "aws",
+  "readURL": "https://YOUR_BUCKET_NAME.s3.amazonaws.com/",
+  "proofsConfig": {
```

Comment:
I'd just leave `proofsConfig` out of the sample entirely. The config file doesn't need to have it, and we should deprecate this proof checking feature in any case.